src/rendering/process/process_exec.ts

```diff
@@ -0,0 +1,39 @@
+import { spawn } from "child_process";
+import { ProcessRenderer } from "./process.js";
+import { ReactNode } from "react";
+
+export async function spawnInProcess(
+  p: ProcessRenderer,
```

Comment:
ahhh … above (in the command files the Renderer is always named `r`, here it’s suddenly `p`

Reply:
Fixed.